Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocCheckBox.css

```diff
@@ -2,39 +2,70 @@
 
 .bocCheckBox
 {
-  margin-right: 0.5em;
+  margin-right: calc(1.25 * var(--remotion-themed-padding));
+  white-space: nowrap;
+}
+
+.bocCheckBox input
+{
+  position: absolute;
+  z-index: -1;
+}
+
+.bocCheckBox > span:last-child
+{
+  display: inline-flex;
+  align-items: center;
+}
```

Comment:
display: flex;
align-items: center;

Reply:
Shouldn't the description `span` stay `inline-flex`?

Reply:
yes, that got proven as stupid post-review.